README.md

```diff
@@ -21,7 +21,8 @@ in order):
 The runner will then try to obtain a login token from the following sources
 (again evaluated in order):
 
-- A `hosts.toml` file stored in one of the following locations:
+- A `credentials.toml` or the deprecated `hosts.toml` file stored in one of the following locations:
```

Comment:
since the  `hosts.toml` file _is_ still supported, perhaps it should still be mentioned here, but explicitly labeled as deprecated? not a big deal

Reply:
@hawkw I'd love to burn our boats on this one and push everyone to the new method. What do you think?

Reply:
I think I must be misunderstanding, the comment does mention that `hosts.toml` is deprecated. Are you thinking a separate line that makes it easier to notice that we're moving away from it?

Reply:
Never mind, I can't read. Carry on!